scene/3d/audio_stream_player_3d.cpp

```diff
@@ -832,7 +832,7 @@ float AudioStreamPlayer3D::get_attenuation_filter_db() const {
 }
 
 void AudioStreamPlayer3D::set_attenuation_model(AttenuationModel p_model) {
-	ERR_FAIL_INDEX((int)p_model, 4);
```

Comment:
This should be `3` instead of `4`. This is very unsafe code. It could easily be broken silently like in this case.

Reply:
Ah never mind. I was looking at my local branch which doesn't have that commit. I still think it's very unsafe code, before and after this change.

Reply:
So that's what you meant by "What if the index type is bigger than int"? I thought you were talking about 2x more capacity of unsigned int...

Reply:
> I thought you were talking about 2x more capacity of unsigned int...

Yes, that's what I meant. This is not related to that.